fix_ruff_all.py

```
@@ -0,0 +1,40 @@
+import subprocess
+import re
+import sys
```

Flake8 (7.3.0)

[error] 'sys' imported but unused

(F401)